### PR TITLE
Improve AMI postprocessing

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**

### CODE REVIEW

File: pyaedt/modules/PostProcessor.py [hunk 1]
```diff
@@ -3783,3 +3783,143 @@ def create_statistical_eye_plot(
...
+        setupname : str
+            Name of the setup.
+        probe_name : str
+            AMI Probe name to use.
+        source_name : str
+            AMI source name to use
+        variation_list_w_value : list
+            List of variations with relative values.
+        unit_interval: float, optional
+            Unit interval in seconds. The default is 1e-9.
```
Comment (MaxJPRey) [inline pyaedt/modules/PostProcessor.py:3811]:
```suggestion
            Unit interval in seconds. The default is ``1e-9``.
```

File: pyaedt/modules/PostProcessor.py [hunk 2]
```diff
@@ -3783,3 +3783,143 @@ def create_statistical_eye_plot(
...
+        ignore_bits: int, optional
+            Number of initial bits to ignore. The default is 0.
```
Comment (MaxJPRey) [inline pyaedt/modules/PostProcessor.py:3813]:
```suggestion
            Number of initial bits to ignore. The default is ``0``.
```

File: pyaedt/modules/PostProcessor.py [hunk 3]
```diff
@@ -3783,3 +3783,143 @@ def create_statistical_eye_plot(
...
+        plot_type : str, optional
+            String containing the report type. Default is all quantity types: ``"InitialWave"``, ``"WaveAfterSource"``,
+             ``"WaveAfterChannel"``,``"WaveAfterProbe"``.
+        clock_tics : list, optional
+            List with clock tics.  Default is take the clock tics from the AMI receiver.
```
Comment (MaxJPRey) [inline pyaedt/modules/PostProcessor.py:3818]:
```suggestion
            List with clock tics.  Default are the clock tics from the AMI receiver.
```

**Review by MaxJPRey — APPROVED** [review]
This is great @Samuelopez-ansys

File: pyaedt/modules/PostProcessor.py [hunk 4]
```diff
@@ -3783,3 +3783,143 @@ def create_statistical_eye_plot(
...
+        clock_tics=None,
+    ):
+        """Sampling a waveform at clock times plus half unit interval.
+
+        Parameters
+        ----------
+        setupname : str
+            Name of the setup.
+        probe_name : str
+            AMI Probe name to use.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:3805]:
```suggestion
            AMI probe name.
```

File: pyaedt/modules/PostProcessor.py [hunk 5]
```diff
@@ -3783,3 +3783,143 @@ def create_statistical_eye_plot(
...
+        source_name : str
+            AMI source name to use
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:3807]:
```suggestion
            Name of the AMI source.
```

File: pyaedt/modules/PostProcessor.py [hunk 6]
```diff
@@ -3783,3 +3783,143 @@ def create_statistical_eye_plot(
...
+        variation_list_w_value : list
+            List of variations with relative values.
+        unit_interval: float, optional
+            Unit interval in seconds. The default is ``1e-9``.
+        ignore_bits: int, optional
+            Number of initial bits to ignore. The default is ``0``.
+        plot_type : str, optional
+            String containing the report type. Default is all quantity types: ``"InitialWave"``, ``"WaveAfterSource"``,
+             ``"WaveAfterChannel"``,``"WaveAfterProbe"``.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:3816]:
```suggestion
            Report type. The default is ``None``, in which case all report types are generated. Options are
            ``"InitialWave"``, ``"WaveAfterSource"``, ``"WaveAfterChannel"``, and ``"WaveAfterProbe"``.
```

File: pyaedt/modules/PostProcessor.py [hunk 7]
```diff
@@ -3783,3 +3783,143 @@ def create_statistical_eye_plot(
...
+        clock_tics : list, optional
+            List with clock tics.  Default are the clock tics from the AMI receiver.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:3818]:
```suggestion
            List with clock tics.  The default is ``None``, in which case the clock tics from
            the AMI receiver are used.
```

**Pull request merged**